gen3release-sdk/gen3release/gith/git.py

```diff
@@ -155,6 +155,7 @@ def create_pull_request_apply(
         the_pr = github_client.create_pull(
             title=pr_title, body=commit_msg, head=branch_name, base="master"
         )
+        the_pr.add_to_labels("gen3-release")
```

Comment:
We dont want to do this - as we use `apply` command while creating PRs for cdis-manifest release PR

Reply:
I think we will have to add an argument for labels and use that in the job 🤔